src/network/network_content.cpp

```diff
@@ -29,6 +29,14 @@
 
 #include "../safeguards.h"
 
+template <typename T>
+T* grow(std::vector<T>& vec, std::size_t size)
```

Comment:
similarly

Reply:
except, this function is only used once - could be inlined?

Reply:
I'll put it in SmallVector for now